components/ReviewContainer.tsx

```diff
@@ -0,0 +1,154 @@
+              Want to Read
+            </button>
+            <p>
+              Little bit about this book, part of the synopsis.Little bit about
+              this book, part of the synopsis.Little bit about this book, part
+              of the synopsis.Little bit about this book, part of the
+              synopsis.Little bit about this book, part of the synopsis.Little
+              bit about this book, part of the synopsis.Little bit about this
+              book, part of the synopsis.Little bit about this book, part of the
+              synopsis.Little bit about this book, part of the synopsis.Little
+              bit about this book, part of the synopsis.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div className="activity-icons flex items-center justify-start">
+        <div className="inline-flex items-center mr-7 text-base">
+          <FaRegHeart className="w-4 mr-2.5"></FaRegHeart> <span>0</span>
+        </div>
+        <div className="inline-flex items-center mr-7 text-base">
+          <FaRegComments className="w-4 mr-2.5"></FaRegComments>
+          <span>0</span>
+        </div>
+        <div className="inline-flex items-center mr-7 text-base">
+          <FaRegBookmark className="w-4 mr-2.5"></FaRegBookmark>
+          <span>0</span>
+        </div>
+      </div>
+      <div className="comment-input-container pl-5 pt-5 hidden">
+        <div className="user-profile flex items-center">
+          <Image
+            src="/user-svg.svg"
+            alt="user image"
+            width={32}
+            height={32}
+            className="rounded-full mx-5 mt-2"
+          />
+          <p className="font-semibold">Jane Doe</p>
+        </div>
+        <input
+          type="text"
+          className="comment-input w-full border-b-2 outline-none mb-0 mx-5 resize-none focus:border-orange-950 transition duration-300 ease-in-out"
+          placeholder="Add a comment..."
+        />
+        <span className="error-message"></span>
+        <button
+          type="submit"
+          className="add-comment-button block ml-auto mr-0 mt-2.5 px-2.5 py-1 border border-orange-950 rounded-md mr-5 bg-transparent hover:bg-orange-950 text-orange-950 hover:text-white transition duration-300 ease-in-out"
+        >
+          Post
+        </button>
+
+        <div className="comment-feed">
+          <article className="comment-container" data-comment-id="">
+            <div className="user-profile flex items-center">
+              <Image
+                src=""
+                alt="user image"
+                width={32}
+                height={32}
+                className="rounded-full mx-5 mt-2"
+              />
+              <p className="font-semibold"></p>
+            </div>
+            <p className="comment-text my-3.5 text-base "></p>
+          </article>
+        </div>
+      </div>
+    </article>
```

Comment:
This seems to big, I would like to see this split in smaller, reusable components